licenses.yaml

```diff
@@ -3093,9 +3093,9 @@ libraries:
 ---
 
 name: Apache Kafka
-version: 2.2.1
+version: 2.2.2
 license_category: binary
-module: extensions/druid-kafka-indexing-service
+module: extensions/druid-kafkakafka-indexing-service
```

Comment:
typo in the module name?

Reply:
hah oops, my search panel must not have opened when searching for other kafkas in this file... actually we need to update the notice section of this file too since this version was released 2019